sale_manual_delivery/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+The goal of this module is to allow the manual creation of delivery slips. If activated,
+a sale order won't direcly impact directly the stock. It will not make a reservation on the stock.
```

Comment:
```suggestion
a sale order won't direcly impact the stock. It will not make a reservation on the stock.
```
This error was already reported at https://github.com/OCA/sale-workflow/pull/404/files#r110530920

Have you made sure all comments in https://github.com/OCA/sale-workflow/pull/404 have been taken into account in the current PR?